### ISSUE TITLE
Support inviting users by multiple emails

### ISSUE THREAD

**Issue by maxceem** [topcoder-platform/tc-project-service#334]
We should support inviting users by multiple emails.

As we cannot do it with a single call as per another issue https://github.com/topcoder-platform/tc-project-service/issues/246

We should update code and use identity service to retrieve data for several emails. To achieve it we should make requests in parallel in batches with a configured number of possible parallel requests.

**Comment by maxceem** [comment]
@vikasrohit we use this endpoint to [check](https://github.com/topcoder-platform/tc-project-service/blob/challenge%2F30095006/src/routes/projectMemberInvites/create.js#L83) if the user which we are inviting be email is an existent user or no. 

What do you think we should do if one of the requests fails?

Should we return to the client that invitation for the particular email failed? Or should we process as if the user is not registered in the system and create an invitation for him together with sending an email with an invitation to register?

**Comment by vikasrohit** [comment]
@maxceem the identity end point is the source of truth for us for validating the existence of a user, so it should not return 404 for an existent user in any case. However, for the worst case scenario, we should treat such cases as new users instead of not failing the invite.

**Comment by maxceem** [comment]
@vikasrohit my main concern is timeout issues. There is a chance that identity service would fail to respond within the timeout.

So in such case, we are not sure if such a user exists or no.

**Comment by vikasrohit** [comment]
Let see how this goes. Anyway we would have the number of parallel requests configurable, so we can tweak it so that we have very less chances of timeouts. And if it turns out that it is no better than single call at at time, we would have no option than to go for member api instead.

**Comment by maxceem** [comment]
@vikasrohit this is done via PR https://github.com/topcoder-platform/tc-project-service/pull/340

**Comment by vikasrohit** [comment]
Verified in dev, closing.
